projects/batfish/src/test/java/org/batfish/main/BatfishTest.java

```diff
@@ -176,6 +180,40 @@ public void testParseTopologyJson() throws IOException {
     assertEquals(topology.getEdges().size(), 2);
   }
 
+  @Test
+  public void testCheckValidTopology() throws IOException {
+    Map<String, Configuration> configs = new HashMap<>();
+    configs.put("h1", createConfiguration("h1", "eth0"));
+    configs.put("h2", createConfiguration("h2", "e0"));
+    //test that checking a valid topology does not throw any exception
+    EdgeSet edges = new EdgeSet();
+    edges.add(new Edge("h1", "eth0", "h2", "e0"));
+    Batfish.checkTopology(configs, new Topology(edges));
+  }
+
+  @Test
+  public void testCheckTopologyInvalidNode() throws IOException {
+    Map<String, Configuration> configs = new HashMap<>();
+    configs.put("h1", createConfiguration("h1", "eth0"));
+    EdgeSet edges = new EdgeSet();
+    edges.add(new Edge("h1", "eth0", "h2", "e0"));
+    _thrown.expect(BatfishException.class);
```

Comment:
similarly, blank line before `_thrown`. Create `topology` before thrown is setup.